### PR TITLE
SRF ID field in predischarge form

### PR DESCRIPTION
**Pull request by sam9111**
#1287
SRF ID filed would be blank now. Since it is an optional field, it doesn't have to be prefilled and can be filled only when the SRF ID has to be updated.

### CODE REVIEW

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -708,8 +708,8 @@ export const PatientHome = (props: any) => {
                         {Array.isArray(patientData.countries_travelled)
                           ? patientData.countries_travelled.join(", ")
                           : patientData.countries_travelled
-                              .split(",")
-                              .join(", ")}
+                            .split(",")
+                            .join(", ")}
```
Comment (smartclash) [inline src/Components/Patient/PatientHome.tsx]: Why was this indentation removed?

File: src/Components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -1150,12 +1150,12 @@ export const PatientHome = (props: any) => {
...
-                        : "") +
-                      (patientData.instituion_of_health_care_worker
-                        ? ", " + patientData.instituion_of_health_care_worker
-                        : "")
+                    (patientData.designation_of_health_care_worker
+                      ? ", " + patientData.designation_of_health_care_worker
+                      : "") +
+                    (patientData.instituion_of_health_care_worker
+                      ? ", " + patientData.instituion_of_health_care_worker
+                      : "")
```
Comment (smartclash) [inline src/Components/Patient/PatientHome.tsx]: And the indentation here?

**Review by smartclash — APPROVED** [review]
LGTM :+1:

**Comment by aparnacoronasafe** [comment]
Firstly, this is already fixed in staging.

Secondly, in preview here, in the discharge pop up window, the date of result comes pre-filled with a random date

**Comment by amruthavijay88** [comment]
Tested, while discharge it's possible to edit/remove SRF ID field. It's also possible to discharge without SRF ID.

**Comment by sam9111** [comment]
> Firstly, this is already fixed in staging.
> 
> Secondly, in preview here, in the discharge pop up window, the date of result comes pre-filled with a random date

chrome-capture


The problem still exists in staging which is what I have fixed and the date appearing is actually the test date.

**Comment by sam9111** [comment]
For some reason, I am not able to make the date blank as well but the srf id is blank on start which is what we need.

of test

**Comment by aparnacoronasafe** [comment]
@sam9111 in the date of test, re do not need the test date appearing, because at discharge this is another test that they are referring to. 

So we don't need pre/filling of date of test. 

Should I file it as a separate issue ?

**Comment by sam9111** [comment]
Ok so it should be blank?

**Comment by sam9111** [comment]
> @sam9111 in the date of test, re do not need the test date appearing, because at discharge this is another test that they are referring to. 
> 
> So we don't need pre/filling of date of test. 
> 
> Should I file it as a separate issue ? 

Yes, I shall try making it blank as well.

Which test is this then? @aparnacoronasafe 

For now, there is only 1 'date_of_test' field in the patient model which is what we are changing here upon discharge.

**Comment by amruthavijay88** [comment]
Tested, SRF ID is blank, however, test's date is getting pre-filled. @sam9111

**Comment by amruthavijay88** [comment]
Tested - both SRF ID and date of test is prefilled now.

**Comment by sam9111** [comment]
> Tested - both SRF ID and date of test is prefilled now.
chrome-capture

**Comment by sam9111** [comment]
> Tested - both SRF ID and date of test is prefilled now.

chrome-capture (1)

**Comment by nihal467** [comment]
test approved

**Comment by bodhish** [comment]
@sam9111 can you fix the merge conflicts

**Comment by sam9111** [comment]
> @sam9111 can you fix the merge conflicts

Done @bodhish

**Pull request merged**